rero/rero-ils
# circulation ui: check if item or patron is in same organisation

* Closes #377

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

How to test:
- `script/boostrap`
- `script/setup`
- Use circulation ui: Check message if item is not in the same organisation
